Makefile

```diff
@@ -150,37 +150,33 @@ else
 	@go test -mod=readonly $(ARGS) $(TEST_PACKAGES)
 endif
 
-.PHONY: run-tests $(TEST_TARGETS)
+cover-html: test-unit-cover
+	@echo "--> Opening in the browser"
+	@go tool cover -html=$(TEST_COVERAGE_PROFILE)
+
+.PHONY: cover-html run-tests $(TEST_TARGETS)
 
 ###############################################################################
 ###                                Linting                                  ###
 ###############################################################################
 
 golangci_lint_cmd=golangci-lint
-golangci_version=v1.49.0
 
 lint:
 	@echo "--> Running linter with revive"
 	@go install github.com/mgechev/revive
-	@revive ./...
+	@revive -config .revive.toml -formatter friendly ./...
 
 lint-fix:
 	@echo "--> Running linter to fix the lint issues"
-	@go install mvdan.cc/gofumpt@latest
-	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(golangci_version)
+	@go install github.com/mvdan/gofumpt
+	@go install github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
Note on `go install`:

A version suffix like `@latest` or `@v1.49.0` will ignore the version of a tool tracked in `go.mod`, but I observed that there were still some `go.sum` changes after `make lint` in practice.

Now the tool versions are pinned in `tools.go` and no `go.sum` instability occurs.